src/client/sandbox/node/window.js

```diff
@@ -95,6 +110,18 @@ export default class WindowSandbox extends SandboxBase {
             return false;
         };
 
+        window.setTimeout = function () {
+            var args = WindowSandbox._processTimerArgs(arguments);
```

Comment:
And I think we should move timers sandbox out of event sandbox and place it in `sandbox/timers.js`, like we do for `cookie` if it's possible.